gro.pro

```diff
@@ -8,10 +8,9 @@
 # A number of variables can be used to modify the compilation:
 #
 #   CONFIG=nogui	Create the command line version of gro
-#   CONFIG=console      Create command line version instead of bundle
```

Comment:
It looks like you branched off of a slightly older version of the master branch.  It would be useful to rebase this off of the current master branch to avoid reverting these changes.  `git rebase master` should do it (you'll get some warnings when you push, so you may need to do `git push --force`).